buidlerConfigV4.js

```diff
@@ -0,0 +1,33 @@
+usePlugin("@nomiclabs/buidler-truffle4");
+usePlugin("@nomiclabs/buidler-web3-legacy");
+
+module.exports = {
+  defaultNetwork: "develop",
+
+  networks: {
+    develop: {
+      url: "http://127.0.0.1:8545",
+      gas: 6000000,
+      timeout: 20000
+    }
+  },
+
+  solc: {
+    version: "0.4.18",
+    optimizer: {
+      enabled: true,
+      runs: 200
+    }
+  },
+
+  paths: {
+    sources: "./contracts",
+    tests: "./test",
+    cache: "./cache",
```

Comment:
The directory used by Buidler to cache its internal stuff. This path is resolved from the project's root. Default value: './cache'.

Since it's the same as the default value, I can remove it.